### PR TITLE
Remove inconsistent tabs from manifest.json files

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes nothing. Everything is meaningless.

### CODE REVIEW

**Comment by pdehaan** [comment]
If you check out the whitespace-less version of the diff (https://github.com/mozilla-services/services-test/pull/47/files?w=0) you can see nothing changed. I just [subjectively] improved spacing and removed mixed tabs and spaces.

I ran all the *.json files through JSONLint and everything was valid, but the mixed indenting was making me sad in my general heart region.

File: autopush/manifest.json [hunk 1]
```diff
@@ -1,35 +1,34 @@
 {
-    "envs" : {
-	"stage": {
-	    "test_types" : [ "stack-check" ],
-            "aws_region": "us-east-1",
-            "stack_label" : "autopush-stage",
-            "aws_region": "us-east-1",
```
Comment (pdehaan) [inline autopush/manifest.json:7]: Added JSON linting via ESLint [**eslint-plugin-json**](https://github.com/azeemba/eslint-plugin-json) module, and it reported this:

```
/Users/pdehaan/dev/tmp/del/services-test/autopush/manifest.json
  7:7  error  Duplicate key 'aws_region'  bad-json
```

**Comment by pdehaan** [comment]
OK, this slightly grew in scope to add basic JSON file linting since it did catch a potential error.

r? @rpappalax

**Comment by rpappalax** [comment]
lots of housecleaning.  thanks for doing it, Peter! +1

**Pull request merged**